crashes.json

```diff
@@ -661,6 +681,27 @@
         }
       ]
     },
+    {
+      "name": "owo",
+      "fix": "To find the solution for SBA's OwO crash, see `-crash sbaowo`. Otherwise, you may be thinking of Kerbybit's MinecwaftOwO, which corresponds to `-crash owomod`.",
+      "causes": [
+        {
+          "method": "regex",
+          "value": "This is a regex test and shouldn't be visible within the Discord server\\."
```

Comment:
Imagine if some dude says that in game

Reply:
crashlogs ≠ chat logs.

ALSO FUCK I GOTTA REVISE THAT SENTENFE GAAIN